src/app/components/bibliography/bibliography.component.html

```diff
@@ -0,0 +1,19 @@
+<div class='biblSpace'>
+  <div class='biblContent'>
+    <span>
+      <span *ngIf="isComplete" class="biblSortingOption">
+        <label> Sort By </label>
+        <ng-select class="w-25 p-3" placeholder="Select" [items]="(sortingField | async)?.sortBy" [clearable]="false"
+          [(ngModel)]="selectSortByField" (change)="sortBy(selectSortByField)" [searchable]="false">
+        </ng-select>
+      </span>
+      <span class="biblSortingDefault">
+        <label class="pr-1 selectLabel"> Alphabetical order</label>
+        <ng-select class="w-25 p-3" placeholder="Select" [items]="(sortingField | async)?.alphOrder" [clearable]="false"
```

Comment:
Make this selected when rendered

Reply:
I do not understand. This is always rendered at creation.

Reply:
Focus on the **selected** part of the sentence.
The ng-select does not show the very first selection as it should.